test/functionalTest/cases/4286_Update_format_for_notification_messages/Update_format_for_notification_messages.test

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'd suggest to add a couple of additional .test files, to check the functionality combined with custom notifications. In particular:

test/functionalTest/cases/4286_update_format_for_notification_messages/update_format_for_notification_messages_custom_no_payload.test

```
# 01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized with httpCustom and no payload field
# 02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedkeyValues with httpCustom and no payload field
# 03. Create entity E1/T1
# 04. Dump accumulator to see notifications
```

test/functionalTest/cases/4286_update_format_for_notification_messages/update_format_for_notification_messages_custom_ngsi_patching.test

```
# 01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized with httpCustom and ngsi field adding a new attribute
# 02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedkeyValues with httpCustom and ngsi field adding a new attribute
# 03. Create entity E1/T1
# 04. Dump accumulator to see notifications
```

(note "update" in lowercase in the .test file names. I'd suggest also to change existing test/functionalTest/cases/4286_Update_format_for_notification_messages/Update_format_for_notification_messages.test to test/functionalTest/cases/4286_update_format_for_notification_messages/update_format_for_notification_messages.test, to keep the lowercase style)

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/70372da52a2a408f76211f4e00046165f5fc743b